packages/rabbitmq/src/amqp/connection.ts

```diff
@@ -393,6 +393,11 @@ export class AmqpConnection {
             throw new Error('Received null message');
           }
 
+          if (msgOptions.routingKey !== msg.fields.routingKey) {
+            channel.nack(msg, false, true);
```

Comment:
This will cause silent infinity loop without any notifications in case any configuration differences beetwen rabbit and routingKey definitions, thant not good. I recommened here - throw some message for handle las in custom errorHandler by this lib or for handle by default error behaviour.